ts/features/payments/home/components/PaymentsHomeEmptyScreenContent.tsx

```diff
@@ -0,0 +1,65 @@
+import {
+  Body,
+  ButtonSolid,
+  H4,
+  IOStyles,
+  IOVisualCostants,
+  Pictogram,
+  VSpacer
+} from "@pagopa/io-app-design-system";
+import React from "react";
+import { View } from "react-native";
+import I18n from "../../../../i18n";
+import { useIONavigation } from "../../../../navigation/params/AppParamsList";
+import { PaymentsBarcodeRoutes } from "../../barcode/navigation/routes";
+
+type Props = {
+  withPictogram?: boolean;
+};
+
+const PaymentsHomeEmptyScreenContent = ({ withPictogram = false }: Props) => {
+  const navigation = useIONavigation();
+
+  const handleOnAddMethodPress = () => {
+    navigation.navigate(PaymentsBarcodeRoutes.PAYMENT_BARCODE_NAVIGATOR, {
+      screen: PaymentsBarcodeRoutes.PAYMENT_BARCODE_SCAN
+    });
+  };
+
+  return (
+    <View
+      testID="PaymentsHomeEmptyScreenContentTestID"
+      style={[
+        IOStyles.flex,
+        IOStyles.alignCenter,
+        IOStyles.centerJustified,
+        { flexGrow: 1, paddingHorizontal: IOVisualCostants.appMarginDefault }
+      ]}
+    >
+      {withPictogram && (
+        <View testID="PaymentsHomeEmptyScreenContentTestID-pictogram">
+          <Pictogram name="empty" />
+          <VSpacer size={16} />
+        </View>
+      )}
+      <H4 style={{ textAlign: "center" }}>
```

Comment:
To get same code style in the code base what do you think to avoid inline style?